includes/apple-exporter/components/class-embed-web-video.php

```diff
@@ -39,6 +41,7 @@ public static function is_embed_web_video( $node, $pattern ) {
 		return (
 			( 'p' === $node->nodeName && preg_match( $pattern, trim( $node->nodeValue ) ) ) // phpcs:ignore WordPress.NamingConventions.ValidVariableName.NotSnakeCaseMemberVar
 			|| ( 'iframe' === $node->nodeName && preg_match( $pattern, trim( $node->getAttribute( 'src' ) ) ) ) // phpcs:ignore WordPress.NamingConventions.ValidVariableName.NotSnakeCaseMemberVar
+			|| ( 'figure' === $node->nodeName && Component::is_gutenberg_embed_figure( $node ) ) // phpcs:ignore WordPress.NamingConventions.ValidVariableName.NotSnakeCaseMemberVar
```

Comment:
Is this necessarily a Gutenberg convention? Does the default embed in the classic editor also do this? We may not want to name this function specifically as `gutenberg` in case this behavior exists in other contexts.

Reply:
Fair point. Will rename this function to `is_embed_figure` to be less specific.